src/PostgREST/ContentTypes.hs

```diff
@@ -0,0 +1,66 @@
+{-# LANGUAGE DeriveAnyClass        #-}
+{-# LANGUAGE DeriveGeneric         #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+
+module PostgREST.ContentTypes
```

Comment:
Looking ahead, I think this should end up in the `Request` namespace somewhere.

Let's move it to `PostgREST.ApiRequest.ContentType` for now?

Reply:
It's also used for response headers and error headers right now, not sure yet where to put it... suggest that we leave it here for now.

Reply:
Ok. For now!! ;)